docs/admin_tutorial.md

```diff
@@ -46,32 +48,389 @@ CREATE EXTENSION pg_diffix;
 ALTER DATABASE test_db SET session_preload_libraries TO 'pg_diffix';
 ```
 
-## Configuration
+## User configuration (per database)
```

Comment:
User configuration is done globally, not per database.

Reply:
I mean the creation of the account and the labelling, the access rights have to be done per database (so better move them to the next section).